applications/luci-app-passwall/luasrc/passwall/util_sing-box.lua

```diff
@@ -1312,7 +1296,7 @@ function gen_config(var)
 				inet4_range = "198.18.0.0/16",
 				inet6_range = "fc00::/18",
 			}
-
+			
```

Comment:
could you remove these whitespaces?

Reply:
Okay. I'll finish fixing it later with the upstream update. Because I see that the upstream has been updated with new content.